test/config.options.test.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const confObj = require('./fixtures/config_files/sample_config.json')
+const { isExcluded } = require('../filter/')
+
+describe('Config options tests', () => {
+  test('Exclude files test', () => {
+    const files = ['tests/func_test.js', 'workflow_test.go', 'src/cmd/main.py']
+    files.forEach(file => {
+      expect(isExcluded(file, confObj.exclude)).toBeTruthy()
+    })
+  })
+
+  test('No excluded files test', () => {
+    const files = ['../tests/', 'workflow.go', 'src/cmd/untilities.py']
+    files.forEach(file => {
+      expect(!isExcluded(file, confObj.exclude)).toBeTruthy()
```

Comment:
As above, let's use `toBe()` rather than `toBe[Truthy|Falsy]()`, furthermore rather than saying this should not `!` be `true`, let's just say it should be false:
```JS
expect(isExcluded(file, confObj.exclude)).toBe(false);
```